Project1/Task10-homework/Collections1.cs

```diff
@@ -0,0 +1,243 @@
+            //Count the total population of all cities and output to the console *
+            Dictionary<string, City> myCity = new Dictionary<string, City>();
+            myCity.Add("Oku", new City(23, 456.5));
+            myCity.Add("Aka", new City(34, 345.4));
+            myCity.Add("mika", new City(24, 355.2));
+            myCity.Add("Lala", new City(44, 445.4));
+            myCity.Add("Saka", new City(54, 555.2));
+            var sorted = myCity.OrderBy(x => x.Value.Area);
+            Console.WriteLine("sorted dictionary by area:");
+            foreach (var item in sorted)
+            {
+                Console.WriteLine(item.ToString());
+            }
+            var reversed = myCity.OrderByDescending(x => x.Value.Population);
+            Console.WriteLine("decending order by population");
+            foreach (var item in reversed)
+            {
+                Console.WriteLine(item.ToString());
+            }
+            double sum = myCity.Sum(v => v.Value.Area);
+            Console.WriteLine($"Sum of area:{sum}");
+        }
+        public static void SortMerge(List<int> i, List<string> s)
+        {
+            i.Sort();
+            s.Sort();
+            s.Reverse();
+            var dic = new Dictionary<int, string>();
+            foreach (var key in i)
+            {
+                foreach (var value in s)
+                {
+                    dic[key] = value;
+                    s.Remove(value);
+                    break;
+
+                }
+            }
+            foreach (var item2 in dic)
+            {
+                Console.WriteLine(item2);
+            }
+        }
+
+        public static void reverseQueue(Queue<int> q)
+        {
+            if (q.Count == 0)
+                return;
+            int fr = (int)q.Peek();
+            q.Dequeue();
+            reverseQueue(q);
+            q.Enqueue(fr);
+        }
+
+        public static void GetMaxValue(Queue<int> q)
+        {
+            Console.WriteLine("Max element:" + q.Peek());
+        }
+
+        public static void DeleteMaxValue(Queue<int> q)
+        {
+            Console.WriteLine("Max element:" + q.Dequeue());
+        }
+
+        public static void SumInt(List<int> list)
+        {
+            int sum = 0;
+            for (int i = 0; i < list.Count; i++)
+            {
+                if (list[i] % 2 == 0)
+                {
+                    sum += list[i];
+                }
+            }
+            Console.WriteLine(sum);
+        }
+
+        public static void Display(List<int> list)
+        {
+            foreach (var item in list)
+            {
+                Console.WriteLine(item);
+            }
+        }
+
+        public static void FindTheWord(List<string> list, int length)
+        {
+            for (int i = 0; i < list.Count; i++)
+            {
+                string a = list[i].ToString();
```

Comment:
why are you trying to call `ToString`, your list is a list of strings, so element in it is string by its own